.github/workflows/release_client.yml

```diff
@@ -0,0 +1,180 @@
+name: Release Client
```

Comment:
This should be merged with https://github.com/crystaldba/crystaldba/blob/main/.github/workflows/release.yml